src/components/ComputeModal.js

```diff
@@ -846,6 +859,9 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
     const gpuCheckboxDisabled = computeExists ? !computeConfig.gpuEnabled : isDataproc(runtimeType)
     const enableGpusSpan = span(
       ['Enable GPUs ', betaVersionTag])
+    const autoPauseCheckboxEnabled = true
+    const enableAutoPauseSpan = span(
+      ['Enable autopause'])
```

Comment:
Should be able to one-line this?

```suggestion
    const enableAutoPauseSpan = span(['Enable autopause'])
```